src/core/fonts.js

```diff
@@ -2463,7 +2463,7 @@ var Font = (function FontClosure() {
       // to be used with the canvas.font.
       var fontName = name.replace(/[,_]/g, '-');
       var isStandardFont = !!stdFontMap[fontName] ||
-        (nonStdFontMap[fontName] && !!stdFontMap[nonStdFontMap[fontName]]);
+        !!(nonStdFontMap[fontName] && stdFontMap[nonStdFontMap[fontName]]);
```

Comment:
it getting hard to read, bu did you mean: 
`!(!nonStdFontMap[fontName] || stdFontMap[nonStdFontMap[fontName]]);`
or `!!(nonStdFontMap[fontName] && !!stdFontMap[nonStdFontMap[fontName]]);` ?